src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -234,6 +235,17 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                     cancelText={translate('common.cancel')}
                     danger
                 />
+                <View style={[styles.pv4, styles.ph5]}>
+                    <ToggleSettingOptionRow
+                        title={translate('common.required')}
+                        switchAccessibilityLabel={translate('common.required')}
+                        isActive={Boolean(policyTagList?.required)}
+                        onToggle={(on) => Tag.setPolicyTagsRequired(policyID, on, route.params.orderWeight ?? 0)}
+                        pendingAction={currentPolicyTag.pendingFields?.required}
+                        errors={currentPolicyTag.errors}
+                        disabled={!Object.values(currentPolicyTag?.tags ?? {}).some((tag) => tag.enabled)}
```

Comment:
If all tags are disabled we won't be able to disable the list either which is incorrect (similar to https://github.com/Expensify/App/pull/42543#discussion_r1622475318)

<img width="380" alt="Screenshot 2024-06-03 at 11 35 48 AM" src="https://github.com/Expensify/App/assets/16493223/19e2027d-1a35-4d21-ada0-4f8e915524ec">

Reply:
https://github.com/Expensify/App/pull/42972/commits/55cd7bbaf2c050dc162997afefafd876a7dc968d as a temporary solution